vmdb/app/models/orchestration_template.rb

```diff
@@ -90,6 +93,16 @@ def validate_content(manager = nil)
     "No #{ui_lookup(:model => 'ExtManagementSystem').downcase} is capable to validate the template"
   end
 
+  def validate_format
+    "Cannot validate format because template type is unknown"
+  end
+
+  def save_with_format_validation
+    msg = draft? ? nil : validate_format
+    raise MiqException::MiqParsingError, msg if msg
```

Comment:
> `validate_format` is actually implemented in subclasses, the error message varies.

K, that makes sense.  See my response to another way to handle the `validate_format` method above...